src/ensembl/src/shared/components/tabs/Tabs.test.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import faker from 'faker';
+import times from 'lodash/times';
+
+import Tabs, { Tab, TabsProps } from './Tabs';
+
+const onTabSelect = jest.fn();
+
+const createTabGroup = (): Tab[] => {
+  const options = times(10, () => ({
+    title: faker.random.words(),
+    isDisabled: faker.random.boolean()
+  }));
+
+  // Make sure we have atleast one enabled and one disabled entry
+  options.push({
+    title: faker.random.words(),
+    isDisabled: true
+  });
+  options.push({
+    title: faker.random.words(),
+    isDisabled: false
+  });
+
+  return options;
+};
+
+const tabsData = createTabGroup();
+
+const tabClassNames = {
+  selected: faker.lorem.word(),
+  default: faker.lorem.word(),
+  disabled: faker.lorem.word(),
+  tabsContainer: faker.lorem.word()
+};
+
+const defaultProps = {
+  tabs: tabsData,
+  selectedTab: tabsData.find((tab) => !tab.isDisabled)?.title || '',
```

Comment:
Why should the `selectedTab` be disabled at all?

Reply:
I'm getting the first tab on the list that is not disabled and assuming it to be the selected tab.